submit_to_challenge.cwl

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env cwl-runner
+#
+# Example score submission file
+#
+cwlVersion: v1.0
+class: CommandLineTool
+baseCommand: synapse
+
+hints:
+  DockerRequirement:
+    dockerPull: sagebionetworks/synapsepythonclient:v2.0.0
```

Comment:
Should we use synapseclient v2.1.0?

Reply:
Ideally, but its not in the dockerhub build for some reason